### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by CindyDorantes**
### Project requirements

You will add tests to the Math Magicians app using [Jest](https://jestjs.io/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). Take a look at their documentation.

- Set up *React Testing Library*.
  - Follow the instructions in the official [documentation](https://testing-library.com/docs/react-testing-library/intro).
  - You don't need to setup Jest (it's included in Create React App).
- Write unit tests for the files `operate.js` and `calculate.js` using Jest.
- Create unit tests for all React components:
  - Use Jest snapshots to test the components.
  - Use React Testing Library to simulate user interaction.
- Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: src/__test__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+test('test function for key AC', () => {
+  const obj = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+  const buttonName = 'AC';
+  expect(calculate(obj, buttonName)).toEqual({ total: null, next: null, operation: null });
+});
```
Comment (Ghiftee) [inline src/__test__/calculate.test.js]: - [ ] According to the requirements, **all the functionalities & scenarios** of the operate.js and calculate.js modules should be covered using Jest. Kindly ensure that you have tests written to cover all the functionalities of calculate.js

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @ijeomaodinko and @CindyDorantes 

Good job working on the project so far! 
✔️ All tests pass
✔️ Well-structured code
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- I suggest you add the second author's name to the README file as both of you contributed to the project.
- I also suggest you add how to run your tests in your README file.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ijeomaodinko** [comment]
Hi, 
The second author's name has been added, the command for the test and all scenarios of testing in calculate applied.

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done implementing the changes requested by the previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**